docs/create-packages/set-package-type.md

```diff
@@ -13,9 +13,9 @@ With NuGet 3.5+, packages can be marked with a specific *package type* to indica
 
 - `Dependency` type packages add build- or run-time assets to libraries and applications, and can be installed in any project type (assuming they are compatible).
 
-- `DotnetCliTool` type packages are extensions to the [dotnet CLI](/dotnet/articles/core/tools/index) and are invoked from the command line. Such packages can be installed only in .NET Core projects and have no effect on restore operations. More details about these per-project extensions are available in the  [.NET Core extensibility](/dotnet/articles/core/tools/extensibility#per-project-based-extensibility) documentation.
+- `DotnetTool` type packages are extensions to the [dotnet CLI](/dotnet/articles/core/tools/index) and are invoked from the command line. Such packages can be installed only in .NET Core projects and have no effect on restore operations. More details about these per-project extensions are available in the  [.NET Core extensibility](/dotnet/articles/core/tools/extensibility#per-project-based-extensibility) documentation.
```

Comment:
Seems fine, but it's a little strange to name package types that we don't own the scenario around. What about "Template" and the SDK package type (forget the name)?

Reply:
The most popular package types appear to be:

Package type | Occurrence
-- | --
DotnetTool | 6590
Template | 4046
DotnetCliTool | 1209
MSBuildSdk | 758
Dependency | 526
AzureSiteExtension | 463
DotnetPlatform | 213

Given this data, it seems that MSBuildSDk, AzureSiteExtensions, and DotnetPlatform are rather esoteric types. I'll add a quick blurb for templates though

Reply:
Is it sustainable for us to keep maintaining this list?

Like what does this list even mean? Is it "package types that Microsoft ships"? Or is it ones that nuget.org recognizes and changes the command palette for?